### PR TITLE
feat(layout): `BlockStatus` add desktop "m" size

### PR DESCRIPTION
**Pull request by vladimirpotekhin**
Fixes #

### CODE REVIEW

File: projects/layout/components/block-status/block-status.component.ts [hunk 1]
```diff
@@ -14,9 +15,13 @@ import {
...
         '[class._card]': 'card',
+        '[attr.data-size]': 'size',
     },
 })
 export class TuiBlockStatusComponent {
     @Input()
     public card = false;
+
+    @Input()
+    public size: TuiSizeL = 'l';
```
Comment (mdlufy) [inline projects/layout/components/block-status/block-status.component.ts:26]: if we can have different size values, maybe add this property to the API tab?

**Review by mdlufy — APPROVED** [review]

File: projects/layout/components/block-status/block-status.style.less [hunk 2]
```diff
@@ -55,3 +55,32 @@ tui-block-status {
...
 }
+
+[data-platform='web'] tui-block-status {
+    &[data-size='m'] {
+        h1,
+        h2,
+        h3,
+        h4,
+        h5,
+        h6 {
```
Comment (mdlufy) [inline projects/layout/components/block-status/block-status.style.less:66]: It looks some cumbersome.. 
but there's probably nothing we can do about it

**Review by mdlufy — APPROVED** [review]

**Review by nsbarsukov — APPROVED** [review]

File: projects/demo/src/modules/components/block-status/examples/6/index.less
[the remark this answers is not in the document]
Comment (vladimirpotekhin) [reply projects/demo/src/modules/components/block-status/examples/6/index.less:1]: Yes, there is a error without empty less

[the remark this answers is not in the document]
Comment (vladimirpotekhin) [reply projects/demo/src/modules/components/block-status/examples/6/index.less:1]: No, my mistake, removed

**Pull request merged**